librz/bin/dwarf/op.c

```diff
@@ -606,10 +606,7 @@ static RzBinDwarfValueType ValueType_from_name(const char *name, ut8 byte_size)
 
 static RzBinDwarfValueType ValueType_from_die(RzBinDwarfEvaluation *eval, const RzBinDWARF *dw, UnitOffset offset) {
 	RzBinDwarfDie *die = ht_up_find(dw->info->die_by_offset, eval->unit->offset + offset, NULL);
-	if (!die) {
```

Comment:
Please remember that `rz_return_val_if_fail()` are removed in the release build. Thus it's better to restore `!die` check